src/middlewares/sentry-middleware.ts

```diff
@@ -13,7 +13,7 @@ function wrappedSentryMiddleware(
     ): Promise<void> => {
       return tracer.span("sentry-middleware", async (span) => {
         try {
-          return withSentry(handler)(request, response);
+          return await withSentry(handler)(request, response);
```

Comment:
Is it the fix you talked about in the PR description ?

Reply:
Nope, I tried this in the past and it did nothing, but after a discussion on #frontend with Fenn, I put it back for the `try/catch`.

The fixes are:
- Add the `return` keyword before the `response.end`.
- Serialising the data and passing it to `response.end` instead of using `response.json`.